packages/preview-middleware-client/src/cpe/init.ts

```diff
@@ -0,0 +1,7 @@
+import log from 'sap/base/Log';
+import type RuntimeAuthoring from 'sap/ui/rta/RuntimeAuthoring';
+
+export default function(rta: RuntimeAuthoring) {
```

Comment:
Don't we need `init(rta)` also here?

Reply:
Nope. `flp/init` is called as part of the initialization of the FLP sandbox. If RTA is enabled and a plugin script is configured then it is called by RTA (not by us) with RTA as parameter. So `cpe/init` is called by RTA already.

Reply:
OK 👍